vignettes/datatable-faq.Rmd

```diff
@@ -26,9 +26,9 @@ h2 {
 }
 </style>
 
-```{r echo=FALSE, file='_translation_links.R'}
+```{r, echo=FALSE, file='_translation_links.R'}
```

Comment:
i imagine this will later trigger a test for un-named chunks, are names supported in this context?